static/src/javascripts/projects/commercial/modules/carrot-traffic-driver.js

```diff
@@ -96,47 +74,18 @@ const desktopRules = {
             minAbove: 400,
             minBelow: 400,
         },
-        ' .element-pullquote': {
-            minAbove: 400,
-            minBelow: 400,
-        },
     },
-    fromBottom: true,
 };
 
 // mobile(320) and above
 const mobileRules = {
```

Comment:
It looks like `desktopRules ` and `mobileRules` are equivalent objects with the same props and values?

Reply:
Good shout, [they are indeed identical](https://editor.mergely.com/E2ygyVc3/).